Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -590,6 +592,18 @@ def add_attribute(event_id: int = None, internal: bool = False, demisto_args: di
     )
 
 
+def convert_to_list(string):
+    try:
+        list_value = ast.literal_eval(string)
```

Comment:
`literal_eval` is not considered a safe function since it allows arbitrary input and has some vulnerabilities related to it (read more about it [here](https://docs.python.org/3/library/ast.html#ast.literal_eval)).